templates/Backend.cpp

```diff
@@ -148,9 +144,9 @@ bool Entity::SendMessage(int msg, const void* data) {
 	{% for message in entity.get_messages_to_handle() %}
 
 		// {{entity.get_type_name()}}'s {{message.get_name()}} message dispatcher.
-		void {{entity.get_message_handler_name(message)}}(Entity* _entity, const void* {% if message.get_num_args() > 0 %} _data{% endif %}) {
+		void {{entity.get_message_handler_name(message)}}(Entity* _entity, const void* {% if message.get_num_args() > 0 %}_data{% else %}/*_data*/{% endif %}) {
```

Comment:
I'm not sure the commented out _data is useful in this case.

Reply:
Alright, removed it again. :)